modules/library/jdbc/src/main/java/org/geotools/jdbc/JDBCDataStore.java

```diff
@@ -1878,7 +2007,7 @@ protected String encodeFID( PrimaryKey pkey, ResultSet rs, int offset ) throws S
         return encodeFID( keyValues );
     }
 
-    protected String encodeFID( List<Object> keyValues ) {
+    protected static String encodeFID( List<Object> keyValues ) {
```

Comment:
This is again more a function than a method. It doesn't need anything from JDBCDataStore instances.